xl2times/transforms.py

```diff
@@ -588,7 +617,8 @@ def process_user_constraint_table(
         # TODO: apply table.uc_sets
 
         # Fill in UC_N blank cells with value from above
-        df["uc_n"] = df["uc_n"].ffill()
+        if "uc_n" in df.columns:
+            df["uc_n"] = df["uc_n"].ffill()
```

Comment:
I definitely still need this check in here for austimes.

Reply:
Yes, we should handle this elsewhere. Will try to create a PR on this soon.

Reply:
@SamRWest I've opened #200 to address this. Any chance you could test whether it resolves the issue with AusTIMES?

Reply:
Thanks :)  I wasn't sure the best place to do this kind of validation.

Reply:
It is still a bit of trial and error, but we should create a DAG for the processing steps soon.